tests/test_structs.py

```diff
@@ -136,3 +136,61 @@ def test_frozen_trial_repr():
                         intermediate_values={})
 
     assert trial == eval(repr(trial))
+
+
+def test_study_summary_study_id():
+    # type: () -> None
+
+    study = optuna.create_study()
+    summaries = study._storage.get_all_study_summaries()
+    assert len(summaries) == 1
+
+    summary = summaries[0]
+
+    assert summary.study_id == summary._study_id
+    with pytest.warns(DeprecationWarning):
+        summary.study_id
+
+
+def test_study_summary_eq_ne_lt():
```

Comment:
It's not immediately obvious where `lt` is tested in this function but can it be omitted from the function name?

Reply:
Sorry for my careless mistake.
Fixed: https://github.com/optuna/optuna/pull/718/commits/472714d88e76231798cb162c3fc99da276ee87ec